src/psconvert.c

```diff
@@ -2424,7 +2424,9 @@ EXTERN_MSC int GMT_psconvert (void *V_API, int mode, void *args) {
 
 		if (GMT->current.setting.run_mode == GMT_MODERN) {
 			if (Ctrl->T.ps) {	/* Under modern mode we can also save the PS file by renaming it */
-				strncpy (out_file, Ctrl->F.file, PATH_MAX-1);
+			        *out_file = '\0'; /* truncate string to build new output file */
```

Comment:
As suggested by @PaulWessel:
```suggestion
			        out_file[0] = '\0'; /* truncate string to build new output file */
```